softdevice_controller/CHANGELOG.rst

```diff
@@ -9,14 +9,6 @@ Changelog
 
 All the notable changes to this project are documented on this page.
 
-Main branch
-***********
-
-Added
-=====
-
-* Central-only and Peripheral-only library variants for the nRF54L Series devices. (DRGN-25081)
```

Comment:
This section might need to be added back later as it is not to be backported but need to be available in main.